chromadb/utils/embedding_functions.py

```diff
@@ -29,6 +27,10 @@
 import json
 import sys
 import base64
+from ..utils.mlx_model import load_model
+import mlx.core as mx
```

Comment:
@Rehan-shah, we tend to avoid direct imports here. Check how the rest of the EFs are implemented with dynamic imports. If we do this, then we make the `mlx` and `transformers` lib a direct dependency. While Mac users are important, they are not the bulk of users.